README.md

```diff
@@ -203,7 +203,7 @@ concern.
 
 ### What programming languages are supported?
 
-Currently SeaGOAT is hardcoded to only process files in the following
+Currently SeaGOAT is hard coded to only process files in the following
```

Comment:
```suggestion
Currently SeaGOAT is hard-coded to only process files in the following
```

That should actually be the correct spelling. But I think even `hardcoded` is better than `hard coded`